src/js/components/Button/doc.js

```diff
@@ -46,7 +46,7 @@ export const doc = Button => {
     color: colorPropType.description(
       'Fill color for primary, label color for plain, border color otherwise.',
     ),
-    disabled: PropTypes.bool
+    disabled: PropTypes.oneOfType([PropTypes.bool, PropTypes.number])
```

Comment:
Think we can remove this now that only a bool is going to get passed to Button

Reply:
yeah I was doing this just to revert back when adding a `disable test`